api/v1alpha1/contentlibraryitemimportrequest_types.go

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ContentLibraryItemImportRequestSource contains the specification of the source for the import request.
+type ContentLibraryItemImportRequestSource struct {
+	// URL is the endpoint that points to an OVF/OVA template that
+	// has to be imported as a new Content Library Item in the
```

Comment:
nit: 
has to -> is to
in the vSphere Content Library. -> in a vSphere Content Library.

Reply:
Updated.